ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -327,10 +338,8 @@ protected void onOverScrolled(int scrollX, int scrollY, boolean clampedX, boolea
       // more information.
 
       if (!mScroller.isFinished() && mScroller.getCurrY() != mScroller.getFinalY()) {
-        int scrollRange = Math.max(
-          0,
-          getChildAt(0).getHeight() - (getHeight() - getPaddingBottom() - getPaddingTop()));
-        if (scrollY >= scrollRange) {
+        int scrollRange = getMaxScrollY();
+        if (scrollY >= getMaxScrollY()) {
```

Comment:
I just meant you're calling getMaxScrollY twice when you already have the result :)